### ISSUE TITLE
Screen flashes to black over and over (network related)

### ISSUE THREAD

**Issue by tergo** [OpenELEC/OpenELEC.tv#495]
Clean install of OpenELEC-ION.x86_64-1.95.2 on Zotac IONITX-B-E (Atom230/Ion). 

Started to add sources (SMB) and configure options. At some point while scraping a network source, the problem started and now it is permanent. 

As soon as it boots to home screen, it just flashes to black (with X cursor) and then back to home screen over and over. Not possible to do anything while this is happening. 

Network related because pulling the network cable stops the problem. But re-connecting it and starting to do some network activity starts it again.

**Comment by construx** [comment]
Ronalde,

Did you ever resolve this problem? I've just performed a fresh install on a Motion Computing M1300 and I'm experiencing the exact same problem. System boots to black screen with x cursor, then flashes between XBMC start screen and black screen with cursor. Any suggestions?

**Comment by fritsch** [comment]
We don't run on such old hardware. If I see the specs right, you have a
single core Pentium 3 and a non supported 830GM Graphics Card. Kodi is an
OpenGL Application ... how should that work in a nice way?

2015-03-20 23:19 GMT+01:00 construx <notifications@github.com>:

> Ronalde,
>
> Did you ever resolve this problem? I've just performed a fresh install on
> a Motion Computing M1300 and I'm experiencing the exact same problem.
> System boots to black screen with x cursor, then flashes between XBMC start
> screen and black screen with cursor. Any suggestions?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenELEC/OpenELEC.tv/issues/495#issuecomment-84168446>
> .
>



-- 
                   Key-ID:     0x1A995A9B
                   keyserver: pgp.mit.edu
==============================================================
Fingerprint: 4606 DA19 EC2E 9A0B 0157  C81B DA07 CF63 1A99 5A9B

**Comment by ronalde** [comment]
construx: I got it running somehow, but really don't remember how ;(. 

However, when I got to run, I wasn't happy with the performance (like updating big libraries and playing back non-hardware supported videos). I switched to a fanless intel core i5 from logic supply, which runs great even without local storage (network boot using pxe/tftp/nfs). So I think fritsch's remark makes perfect sense.

Regards and good luck with your setup.
